model-engine/model_engine_server/infra/gateways/k8s_resource_parser.py

```diff
@@ -2,9 +2,6 @@
 import re
 from typing import Union
 
-MAX_CONCURRENCY_TO_TARGET_CONCURRENCY_RATIO = 2.0
```

Comment:
Wouldn't this technically change our autoscaling sensitivity? cc @seanshi-scale

Reply:
Which might be fine, just being aware of production behavior changes.

Reply:
yes this would but i think people are not aware of this ratio at all (like users specify per worker concurrency target to be 10 and HPA uses 5)